### DISCUSSION TITLE
how do i enable player controls i3lock

### DISCUSSION THREAD

**Discussion by KiingWatermelon** [i3/i3#4997] [Q&A] [answered]
i have this in my i3 config
`bindsym XF86AudioPlay exec --no-startup-id playerctl play-pause && $refresh_i3status`
and it works when im using i3 but when i open i3lock i cant play pause or skip songs
is there a way to enable these hotkeys in i3lock?
sorry if this is the wrong place to ask

**Comment by Airblader** [comment, marked as answer]
i3lock tries to block all keyboard input. There's no way to allowlist certain keys.

**Reply by KiingWatermelon** [reply]
oh ok
thank you
